.bazelrc

```diff
@@ -149,6 +149,12 @@ test:webdriver-debug --test_env=DISPLAY
 # otherwise), and display verbose webdriver output in the terminal.
 test:webdriver-debug --test_output=streamed
 
+# Currently only works for go tests
+# Coverage outputs could be viewed with `genhtml`:
```

Comment:
I also recently added support in the UI to view this on the test results page

Reply:
Yup, however I think it's rendering using the wrong version of the source file 🤔 
Perhaps it was because I did not push the changes to github just yet?

Reply:
Yeah, if you push the changes to Github it should be able to render correctly